android/app/build.gradle

```diff
@@ -83,6 +84,15 @@ android {
   }
 }
 
+devicefarm {
```

Comment:
Ahh, right.  What if the project and device pool names were also set by environment variables? Would that be safe to try?

Reply:
I think that would be better, but it would best to either have this code live somewhere else or somehow remove it when generating the expokit template. Would rather not continue to clutter up other people's project files unnecessarily.

Reply:
Does the top-level gradle file also get copied?  If not, is it possible to add arbitrary configuration in a way that would apply to this specific sub-project?

Reply:
It does; here's the full list of files that get copied to the expokit template:
https://github.com/expo/expo/blob/1497d15e12b30950126ce0fdf8714df5d185c88b/tools/expotools/src/ExpoKit.ts#L57-L132

Reply:
Ah.  No problem, in that case, I can go back to my first approach, of fastlane using awscli and curl.

Reply:
I changed this block to be configured by environment variables, as we discussed.  Downstream expokit projects could ignore it, delete it, or set the variables for their own projects.